docs/firebase.md

```diff
@@ -247,33 +222,50 @@ look something like the following example:
 ```ts
 const googleAIapiKey = defineSecret("GOOGLE_GENAI_API_KEY");
 
-export const generatePoem = onFlow(
-  ai,
+const generatePoemFlow = ai.defineFlow(
   {
     name: "generatePoem",
     inputSchema: z.string(),
     outputSchema: z.string(),
-    authPolicy: firebaseAuth((user, input) => {
-      if (!user.email_verified) {
-        throw new Error("Verified email required to run flow");
-      }
-    }),
-    httpsOptions: {
-      secrets: [googleAIapiKey],
-      cors: '*',
-    },
   },
   async (subject: string) => {
     const { text } = await ai.generate(`Compose a poem about ${subject}.`);
     return text;
   }
 );
+
+export const generatePoem = onCallGenkit(
+  {
+    authPolicy: isSignedIn(),
+    cors: '*',
+    secrets: [googleAIapiKey],
+  },
+  generatePoemFlow
+);
+```
+Since you define both the Genkit flow and Cloud function when using
+`onCallGenkit`, you can visit the Developer UI to test your flow locally.
+
+```posix-terminal
+cd $PROJECT_ROOT/functions
+
+npx genkit start -- npx tsx --watch src/index.ts
+```
+
+or
+  
+```posix-terminal
+cd $PROJECT_ROOT/functions
+
+npm run genkit:start
 ```
 
+You can now navigate to the URL printed by the `genkit start` command to access.
+
```

Comment:
I moved this from an optional section below. I feel like this part is not relevant anymore and we can remove it from these docs.